spec.emu

```diff
@@ -70,18 +69,14 @@ contributors: Richard Gibson
             </tbody>
           </table>
         1. Set _product_ to the string-concatenation of _product_ and _abbrev_.
-      1. Else if _C_ has a numeric value less than 0x0020 (SPACE), then
+      1. Else if _C_ has a numeric value less than 0x0020 (SPACE)<ins>, or _C_ is a <emu-xref href="#leading-surrogate"></emu-xref> or <emu-xref href="#trailing-surrogate"></emu-xref></ins>, then
```

Comment:
Picky but also quite fair.  I've massaged the wording around to "or _C_ has the same numeric value as a leading-surrogate code unit or trailing-surrogate code unit".  It's a little awkward because I have to apply manual type conversions to both left-hand and right-hand sides, but maybe it does the trick?